plugins/poll/assets/stylesheets/common/poll.scss

```diff
@@ -25,9 +25,16 @@ div.poll {
   }
 
   img {
+    // TODO: remove once disable_image_size_calculations is removed
     // needed to override internal styles in image-sizing hack
     max-width: 100% !important;
     height: auto;
+    // Hacky way to stop images without width/height
+    // from causing abrupt unintended scrolling
+    &:not([width]):not([height]) {
+      height: 200px !important;
```

Comment:
Is the width fixed somewhere as well? I don't think the `max-width` above is enough to prevent jumping?

Reply:
In my testing, only the height was necessary. Regardless of the width, the browser will reserve that specific height for the asset. And as far as scrolling is concerned, only the height is important. 

I think not setting width also means that an image with width=500 and height=200 won't need to be shrunk to fit a 200x200 box.

Reply:
> And as far as scrolling is concerned, only the height is important.

Unfortunately the width also matters, because a wider image can force other elements (e.g. text) to wrap, thereby taking up more height.

Reply:
Right, I hadn't thought of inline images. Ok let's try setting width too.